### ISSUE TITLE
trouble with the latest version.

### ISSUE THREAD

**Issue by needwalter** [KaiKikuchi/QuickShop#159]
Hi!
firstly i whant to thanks you very much for your plugins! 
But... since i upgrade my server i can't use it anymore.
I have drop the .jar in my new serv. and... the plugins seems so be working well, until i try to create a shop..
when it say " write a price in the tchat"  i  can't do it... 
I give u my starting log in a pastbin:  https://pastebin.com/uEuX7R6j
Do you thinks you can help me ? or give me youre neawest version of the plugins ?

Thank you very much for your answer ! 

Walter

**Comment by KaiKikuchi** [comment]
Your chat plugin may be causing QuickShop not being able to get the chat input.

**Comment by needwalter** [comment]
what do you thinks i can do to fix that ? or somethings i can change ?
I really like your plugins  and i really don't want to use another bad one...
Thanks you very much for your help and your answer 

Walt'

**Comment by KaiKikuchi** [comment]
Can you try using EssentialsX instead of Essentials?

**Comment by needwalter** [comment]
i'm already using essentialsX ....  what can i do more ?

**Comment by KaiKikuchi** [comment]
I am using EssentialsX 2.0.1, and it works fine. Please try again, and post the server log part when you actually try to make a shop.

**Comment by needwalter** [comment]
well there is somethings else you need to know.... i'm running bungee serv, so i use BungeetChat plugin to link all my servers...
When you need to sell or buy somethings , you need to write it on the tchat... but i thinks the tchat is going through the proxy not in the "local" server... 
How can i fix that ? do you thinks it's the probleme ?

**Comment by KaiKikuchi** [comment]
Yes it is probably caused by your Bungeecord chat plugin that blocks the messages. You can use the "/qs create" command while looking at the chest.

**Comment by needwalter** [comment]
but when you whant to buy from a chest ... you still need to write the number of item you whant to buy .... isn't  it ?
